src/order/models/order.request.ts

```diff
@@ -0,0 +1,7 @@
+import { OrderItemRequest } from "./orderItem.request";
+
+export class OrderRequest{
```

Comment:
I believe there should be a space before {
What's the behaviour of default formatter?